rice-middleware/impl/src/main/java/edu/arizona/rice/kim/service/impl/UaUiDocumentServiceImpl.java

```diff
@@ -3,7 +3,7 @@
  * Copyright 2009 State of Arizona Board of Regents
  *
  */
-package edu.arizona.kim.service.impl;
+package edu.arizona.rice.kim.service;
```

Comment:
Package name should also be changed to be edu.arizona.rice.kim.service.impl.
One more thing, I've looked in the Spring wiring for the UiDocument Service and I didn't see any commit to use the UaUiDocumentService.... the only I've found is in rice-middleware/...KimWebSpringBeans.xml and it's using the old one:
 `<bean id="kimUiDocumentService" class="org.kuali.rice.kim.service.impl.UiDocumentServiceImpl" />`
Are we missing in this commit the Spring Injection to the new Ua service?